.github/workflows/main.yml

```diff
@@ -172,3 +172,45 @@ jobs:
       - name: Run tests
         run: |
           pytest -m 'vetiver'
+  
+  test-jupyter:
+    runs-on: ubuntu-latest
+    env: 
+      CONNECT_LICENSE: ${{ secrets.RSC_LICENSE }}
+      ADMIN_API_KEY: ${{ secrets.ADMIN_API_KEY }}
+    steps:
+      - uses: actions/checkout@v2
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - uses: extractions/setup-just@v1
+      - name: Build Containers
+        run: |
+            just integration-testing/build
+      - name: Start Connect + rsconnect-jupyter
+        run: |
+            just integration-testing/up
+        
+      - name: Run Cypress Tests
+        run: |
+            export ADMIN_API_KEY="${{ secrets.ADMIN_API_KEY }}"
+            just integration-testing/up-cypress
+
+      # Videos are captured whether the suite fails or passes
+      - name: Save videos
+        uses: actions/upload-artifact@v3
+        if: always()
+        with: 
+          name: cypress-videos
+          path: integration-testing/cypress/videos
+          if-no-files-found: ignore
+          retention-days: 1
```

Comment:
Where are these files uploaded? Do I need to request access to cypress.io from IT?

Reply:
They're uploaded as artifacts.  You can find them at the bottom of the page:
https://github.com/rstudio/rsconnect-python/actions/runs/4135994847

Reply:
Very cool! Thank you!